WindowsServerDocs/storage/refs/refs-overview.md

```diff
@@ -59,27 +59,48 @@ ReFS is designed to support extremely large data sets--millions of terabytes--wi
 
 ## Supported deployments
 
+Microsoft has developed NTFS specifically for general-purpose use with a wide range of configurations and workloads, however for customers specially requiring the availability, resiliency, and/or scale which ReFS provides, Microsoft supports ReFS for use under the following configurations and scenarios. 
```

Comment:
Should be "...scale that ReFS provides..."

Reply:
Agreed.